src/pages/index.tsx

```diff
@@ -103,39 +103,41 @@ export const App = () => {
     !queryResult.data?.hits?.total["value"];
 
   return (
-    <div
-      className={`${
-        isHomePage ? "relative" : ""
-      } bg-custom-background text-custom-primary-text`}
-    >
-      <main
-        id="main"
-        className="min-h-[95dvh] flex w-full items-center pt-[60px] md:pt-[76px] 2xl:pt-[122px]"
+    <>
```

Comment:
remove the react fragment, there should be no changes on this file related to adding boss banner